packages/w/wireless-regdb/monitoring.yml

```diff
@@ -1,6 +1,6 @@
 releases:
   id: 15257 # Check https://release-monitoring.org/
```

Comment:
For this file 15257 is correct, so please delete the rest of the line.

I can't find an rss feed, so leave `rss: ~` and delete the rest of the line